app/cafe.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        # Check if visitor is vaccinated
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(
+                f"{visitor.get("name", "Visitor")}"
+                f" is not vaccinated and cannot enter {self.name}.")
+
+        # Check if the vaccine is expired
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date and expiration_date < datetime.date.today():
+            raise OutdatedVaccineError(
+                f"{visitor.get("name", "Visitor")}'s"
+                f" vaccine is expired and cannot enter {self.name}.")
+
+        # Check if visitor is wearing a mask
+        if not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError(
+                f"{visitor.get("name", "Visitor")}"
+                f" is not wearing a mask and cannot enter {self.name}.")
```

Comment:
You are using both single and double quotes in your code, which is against the Code Style guidelines in the checklist. Please stick to one type of quotes, preferably double quotes.